openid-federation-1_0.xml

```diff
@@ -761,9 +761,193 @@
+	      <t>
+		The Entity Statement MUST be a signed JWT.
+	      </t>
+	      <t>
+		The Entity Statement MUST have a
+		<spanx style="verb">typ</spanx> header parameter with the value
+		<spanx style="verb">entity-statement+jwt</spanx>.
+	      </t>
+	      <t>
+		The Entity Statement MUST have an
+		<spanx style="verb">alg</spanx> (algorithm) header parameter
+		with a value that is an acceptable JWS signing algorithm;
+		it MUST NOT be <spanx style="verb">none</spanx>.
+	      </t>
+	      <t>
+		The Entity Identifier of the Entity to which the Entity Statement refers
+		MUST match the value of the
+		<spanx style="verb">sub</spanx> (subject) Claim.
+	      </t>
+	      <t>
+		The Entity Statement MUST have an
+		<spanx style="verb">iss</spanx> (issuer) Claim
+		with a value that is a valid Entity Identifier.
+	      </t>
+	      <t>
+		When the <spanx style="verb">iss</spanx> (issuer) Claim value
+		matches the <spanx style="verb">sub</spanx> (subject) Claim value,
+		then the Entity Statement is this Entity's Entity Configuration.
+		When they do not match, the Entity Statement is a Subordinate Statement.
+		When the Entity Statement is a Subordinate Statement,
+		the <spanx style="verb">iss</spanx> Claim value MUST match
+		one of the values in the
+		<spanx style="verb">authority_hints</spanx> array
+		in the Entity Configuration for the Entity whose Entity Identifier
+		is the value of the <spanx style="verb">sub</spanx> Claim;
+		otherwise the Federation graph is not well-formed.
+	      </t>
+	      <t>
+		The current time MUST be after the time represented by the
+		<spanx style="verb">iat</spanx> (issued at) Claim
+		(possibly allowing for some small leeway to account for clock skew).
+	      </t>
+	      <t>
+		The current time MUST be before the time represented by the
+		<spanx style="verb">exp</spanx> (expiration) Claim
+		(possibly allowing for some small leeway to account for clock skew).
+	      </t>
+	      <t>
+		The <spanx style="verb">jwks</spanx> (JWK Set) Claim
+		MUST be present, with a value that is a valid
+		JWK Set <xref target="RFC7517"/>.
+	      </t>
+	      <t>
+		Obtain the Entity Configuration for the issuing Entity -
+		the Entity with the Issuer Identifer found in the Entity Statement's
+		<spanx style="verb">iss</spanx> (issuer) Claim.
+		When the <spanx style="verb">iss</spanx> and
+		<spanx style="verb">sub</spanx> Claim values match, this is
+		the Entity Statement being validated itself.
+		Otherwise, this can be obtained either from a Trust Chain or
+		by retrieving it as described in
+		<xref target="federation_configuration"/>.
+	      </t>
+	      <t>
+		The Entity Statement's
+		<spanx style="verb">kid</spanx> (Key ID) header parameter value
+		MUST be a non-zero length string and
+		MUST exactly match the <spanx style="verb">kid</spanx> value
+		for a key in the <spanx style="verb">jwks</spanx> (JWK Set) Claim
+		of the Entity Configuration of the issuing Entity.
+	      </t>
```

Comment:
To better support key rollover, replace 837 to 843 with:

The Entity Statement's <spanx style="verb">kid</spanx> (Key ID) header parameter value
MUST be a non-zero length string and
MUST exactly match the <spanx style="verb">kid</spanx> value
for a key in the <spanx style="verb">jwks</spanx> (JWK Set) Claim
of the Entity Configuration of the issuing Entity.
The <spanx style="verb">jwks</spanx> Claim MAY contain multiple active signing keys
(for example, during key rotation). A validator MUST accept any signature
that verifies successfully with a currently published key in that set.
Publishers SHOULD retain the retiring key alongside the new key
for an overlap period sufficient to allow dependent Entity Statements
to refresh before the old key is withdrawn.

Reply:
The validation section isn't the place to add statements about how the "jwks" claim is used.  If you want to file an issue suggesting text to add in the claim definition, I'd welcome that.